### PR TITLE
added calculator.js file

### PR DESCRIPTION
**Pull request by kzvm**
I'm adding a js file with math operations for lesson02 assignment

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// JavaScript source code
+let y = 42;
+let x = 2;
+let z = 3;
+let k = y / (x * z);
+console.log(k);
+console.log(k + y);
+console.log(k / x);
+
```
Comment (MadEste) [inline calculator.js:9]: 👍

File: countdown.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+let x=3
+
+while (x>-1) {
+  console.log(x);
+  x--;
+}
```
Comment (MadEste) [inline countdown.js:6]: 👍

File: exercise-two.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+let surf = ['I', 'love', 'surfing', 'in', 'my', 'home', 'town'];
+
```
Comment (MadEste) [inline exercise-two.js:2]: I think you had some misunderstanding of what the assignment required. You should have created sentence string and converted it to an array:
`let surf = 'I love surfing in my home town.';`
`let surfArray = surf.split(' ');`

**Comment by kzvm** [comment]
I think, I was able to fix the error, thanks

https://github.com/kzvm/html200-coding

https://github.com/UWFront-End-Cert/html200-coding/pull/33


let surf = 'I love surfing in my home town.';

const surfing = surf.split(' ');

console.log(surfing);
console.log(surfing[2]);

File: lesson4Assig1.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+function numberSquared() {
+    let input = prompt('Enter a number to be squared')
```
Comment (MadEste) [inline lesson4Assig1.js:2]: Nice work! You might also want to use the Number() function around prompt to make sure the input is stored as a number and not a string.

File: lesson5Assig1.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+const D = 'D';
+const B = 'B';
+
+
+let total = 0;
+
+function typeAction(action) {
+    let input = prompt('Type Q, W, D or B');
+
+    while (input !== 'Q' || input !== 'q') {
```
Comment (MadEste) [inline lesson5Assig1.js:12]: Good job checking for upper and lowercase

File: lesson5Assig1.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
+const Q = 'Q';
```
Comment (MadEste) [inline lesson5Assig1.js:1]: These const variables are not required as you do not use them elsewhere in the code.

**Pull request closed without merging**